daprdocs/content/en/developing-applications/integrations/cloud-providers/authenticating-azure.md

```diff
@@ -0,0 +1,407 @@
+---
+type: docs
+title: "Authenticating to Azure"
+linkTitle: "Authenticating to Azure"
+description: "How to authenticate Azure components using Azure AD and/or Managed Identities"
+aliases:
+  - "/operations/components/setup-secret-store/supported-secret-stores/azure-keyvault-managed-identity/"
+  - "/reference/components-reference/supported-secret-stores/azure-keyvault-managed-identity/"
+---
+
+## Common Azure authentication layer
+
+Certain Azure components for Dapr offer support for the *common Azure authentication layer*, which enables applications to access data stored in Azure resources by authenticating with Azure AD. Thanks to this, administrators can leverage all the benefits of fine-tuned permissions with RBAC (Role-Based Access Control), and applications running on certain Azure services such as Azure VMs, Azure Kubernetes Service, or many Azure platform services can leverage [Managed Service Identities (MSI)](https://docs.microsoft.com/azure/active-directory/managed-identities-azure-resources/overview).
+
+Some Azure components offer alternative authentication methods, such as systems based on "master keys" or "shared keys". Whenever possible, we recommend authenticating your Dapr components using Azure AD for increased security and ease of management, as well as for the ability to leverage MSI if your app is running on supported Azure services.
```

Comment:
```suggestion
Some Azure components offer alternative authentication methods, such as systems based on "master keys" or "shared keys". Whenever possible, it is recommended that you authenticate your Dapr components using Azure AD for increased security and ease of management, as well as for the ability to leverage MSI if your app is running on supported Azure services.
```

Reply:
This one can be simplified to "Whenever possible, use Azure AD to authenticate your Dapr components, for increased security..."